index.html

```diff
@@ -36,29 +36,37 @@ <h1>webp-hero</h1>
 		<img alt="" src="data:image/webp;base64,UklGRhgFAABXRUJQVlA4WAoAAAAgAAAANgAAJwAASUNDUBgCAAAAAAIYAAAAAAQwAABtbnRyUkdCIFhZWiAAAAAAAAAAAAAAAABhY3NwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAQAA9tYAAQAAAADTLQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAlkZXNjAAAA8AAAAHRyWFlaAAABZAAAABRnWFlaAAABeAAAABRiWFlaAAABjAAAABRyVFJDAAABoAAAAChnVFJDAAABoAAAAChiVFJDAAABoAAAACh3dHB0AAAByAAAABRjcHJ0AAAB3AAAADxtbHVjAAAAAAAAAAEAAAAMZW5VUwAAAFgAAAAcAHMAUgBHAEIAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAFhZWiAAAAAAAABvogAAOPUAAAOQWFlaIAAAAAAAAGKZAAC3hQAAGNpYWVogAAAAAAAAJKAAAA+EAAC2z3BhcmEAAAAAAAQAAAACZmYAAPKnAAANWQAAE9AAAApbAAAAAAAAAABYWVogAAAAAAAA9tYAAQAAAADTLW1sdWMAAAAAAAAAAQAAAAxlblVTAAAAIAAAABwARwBvAG8AZwBsAGUAIABJAG4AYwAuACAAMgAwADEANlZQOCDaAgAA8AsAnQEqNwAoAD8RcrFRrCakoq1JIYAiCWMAxQvzKWvMWEBV0O83F6qTtffTCUCjZAdKg0wGWOqBCO/y7i1ZTvpOb73EJUd1NWH6+bhJGc13cl8Ek8gLKri54h2i6nzCCxlpIE0kEAAA/pGaLUHMdU1S+L11hVyk+eGRFICWbNctrYo6YBmCUW6tozZSkXyqfl0uj3aF3OW6flxRPPX2dM/A2zRGaRtYeiI32oWXnYeQmUJZb22fkkd3UlJR08pBfCLw1PdUcfYp3PHZRxikInPHx4p5vSlhYTep4uyBTER+0xDi+Z4+GW63NOAjvdVyfI9ChxzePUbWM5OTGSkK8K+pR3B1qpXPi/P8XwfWBHH+bvOBW0JMeapJoR1Bhx8paKGYzu/14H8fhHeRqVIbUrZjRlS224BpVrusC/tzYpRJn6vrdTPj69HVCcIa1rZP0HKnWQT5U98xToaPO9FWkbj51uDnNaV4rFncZXEnHumG22NEXCA+Ix8UhE/2i3nfwC0zlSuD57dLohevPDGCw18Kpd/PwFTtzAzMIE1QMN//kSNA2HpacgXTR7WEo5qCzzyoNiLp/HGDrAtIOXzTcB1bbrNvmNJPxwxDT2QmMUYEh8lmPQzZzwTPllT2uoMhN3qhapMiujQjXvZgkADfHnffhT2IhIICjcNCQ46sOTJn+Tf0Gyq33Z2aE4P3jezb31mTufFcfTDPMLsb73E9pX5gd1Rezi98clelNuuHYNd8taps+p+fIcvrmY+iTFV7tTzCrfZUfLDTRqF2+Oeat6cidqjD/xCI29A/Wi0+x6KVvn5GPC0Xlmsvvh2bPcWhbvwz0It3zbXCyx09E1F1gekihEJfzr8Sr2lrxfIQcGMxVdwu6wOKZI5hQ/plxJyD6Aql6H8JVAIKxlSxUp5IAN1hkUBBDEtqFRX9FTGrEP/kLdF1DoiSg+WuQal4079NJpB9jYyMUaAAAA=="/>
 
 		<script src="dist-cjs/polyfills.js"></script>
```

Comment:
Is `polyfills.js` required? can it also be conditionally loaded or maybe built into `webp-hero.bundle.js` directly?

Ideally we would only need to enqueue `conditionally.js` to get it to work, and if the browser already supported WebP nothing else would load.

Reply:
the polyfills are required by IE11 and older, and surely other old browsers too.

you raise a good point!

i think a good solution might be a simple new attribute, `data-include-polyfills`, which would cause *conditionally* to load the polyfills prior to the webp-hero bundle.

Reply:
👍🏼 that sounds like a good option. It would also be helpful to document exactly when the polyfills are needed, maybe a list of specific browsers that require it? I could help with a follow up task and log that info while doing browserstack testing

Reply:
excellent idea, i'll try to generate a list of which browsers benefit from the polyfills for the readme :+1: